src/python_pachyderm/pfs.py

```diff
@@ -36,9 +41,11 @@ def from_pb(commit: pfs_pb2.Commit) -> "Commit":
             branch=commit.branch.name,
             id=commit.id,
             repo_type=commit.branch.repo.type,
+            project=commit.branch.repo.project.name,
         )
 
 
+# TODO: How to reconcile tuple commit format with projects.
```

Comment:
This is a tough one. I'm leaning towards `(repo, brach-or-commit-id, project, commit-id, repoType)`. We should keep repo and branch-or-commit-id as the first two as they're necessary and sufficient. To me, projects feels like it would be more referenced than commit-id, but again, that's just a guess.